underpants.js

```diff
@@ -194,7 +210,7 @@ var _ = {};
 */
 
 
-/** _.pluck()
+/** _.pluck
```

Comment:
I think pluck should move to the top before filter if we want to keep all of the functions/higher order functions together.

Reply:
The reason for pluck being where it is is that we want students to use _.map() to solve it so it should follow map. However, the tests currently don't require them to do so. I can change the language in the description to ask that if they solved it without _.map to do so as a bonus challenge?

Reply:
I think we should add the test to make sure they are using _.map, it's pretty important for people to understand functional programming with the use of functions they just built.